docs/tests.md

```diff
@@ -49,6 +49,12 @@ PostgreSQL database backend.
 Further configuration options can be found here
 [Pytest configuration](https://docs.pytest.org/en/7.1.x/reference/customize.html).
 
+You can specify additional pytest arguments to the make targets by using the
+environmental variable `EXTRA_PYTEST_ARGS`:
+```sh
+EXTRA_PYTEST_ARGS='-k test_source_suppress_export'  TEST=tests/functional/suppress make test_analyzer_feature
```

Comment:
Could you please add one more sentence about the purpose of this `-k` flag? I think this is a very useful knowledge that this is the way of running a specific test function. Thanks!